Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -98,7 +102,18 @@ public void setTargetSpeedForRange(double rangeInInches) {
     }
     
     public boolean isWheelAtSpeed() {
-        return super.getSpeed() >= super.getTargetSpeed() * wheelSpeedThresholdPercentage.get();
+        // Speed is in native units and can be negative
+        double currentSpeed = Math.abs(super.getSpeed());
+        if (currentSpeed <= 1) {
```

Comment:
Ah, so the issue is that the target speed is zero when the agitators aren't requested to be moving, and that triggers the feeder. As I see it, the fix should be in the command that runs the feeders; this method is not solely used for that purpose.